tetris.py

```diff
@@ -29,6 +30,7 @@ def run_gui():
 
     tetris_gui = TetrisGUI(game_speed, tetris_canvas)
     tetris_gui.tetris_game.new_block()
+    tetris_gui.tetris_game.full_line_clear()
```

Comment:
This is not necessary. Delete this (and to avoid getting things like this into your pull requests, pull on main before making a new branch).